.github/workflows/pushimage-next.yml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Next Dockerimage
+
+on:
+  push:
+    branches: [main]
+  repository_dispatch:
+    types: [build]
+
+jobs:
+  registryViewerBuild:
```

Comment:
jobs names should follow kebab case and should have a display name
i.e.
```yaml
jobs:
  registry-viewer-build:
    name: Registry Viewer Build
```

Reply:
Changes applied.